cmake/buildinfo/buildinfo.cmake

```diff
@@ -0,0 +1,29 @@
+cmake_host_system_information(RESULT CPU_NAME QUERY PROCESSOR_NAME)
+cmake_host_system_information(RESULT CPU_DESC QUERY PROCESSOR_DESCRIPTION)
+
+add_custom_command(
+	OUTPUT cmake_info.json
+	COMMAND ${CMAKE_COMMAND} -E capabilities >> cmake_info.json
+	COMMENT "Generating ${NBL_ROOT_PATH_BINARY}/cmake_info.json"
```

Comment:
since it's a script, you must check if `NBL_ROOT_PATH_BINARY` is defined otherwise error